### ISSUE TITLE
Add Docker instructions to wiki

### ISSUE THREAD

**Issue by billsacks** [ESCOMP/CTSM#1288]
We should put instructions like this on the wiki

https://docs.google.com/document/d/1EUb1YTsWvG0Q2eH7fs9H2fC7P_xR5BG-WyL7R9c8_aU/edit

**Comment by glemieux** [comment]
As additional reference, we have some wiki documentation for running the ngeetropics docker images: https://github.com/NGEET/docker-fates-tutorial/wiki/Building-and-running-cases

**Comment by wwieder** [comment]
@samsrabin is this no longer needed, now that you've moved this into the documentation itself?

**Comment by samsrabin** [comment]
Good call; the instructions for Mac and Windows are linked in [this section](https://escomp.github.io/CTSM/users_guide/working-with-documentation/docs-intro-and-recommended.html#building-the-docs) of the User's Guide. Closing.
